### PR TITLE
[RLlib] APPO config objects

### PR DESCRIPTION
**Pull request by sven1977**
APPO config objects

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: rllib/__init__.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def setup(self, config):
 
 _setup_logger()
 
-usage_lib.record_library_usage("rllib")
+# usage_lib.record_library_usage("rllib")
```
Comment (maxpumperla) [inline rllib/__init__.py:60]: Could you please explain this? Can we remove it altogether?

File: rllib/agents/ppo/appo.py [hunk 2]
```diff
@@ -23,69 +23,140 @@
...
-    _allow_unknown_configs=True,
-)
 
-# __sphinx_doc_end__
-# fmt: on
+class APPOConfig(impala.ImpalaConfig):
+    """Defines a A2CTrainer configuration class from which a new Trainer can be built.
+
+    Example:
+        >>> from ray import tune
```
Comment (maxpumperla) [inline rllib/agents/ppo/appo.py:34]: APPOConfig import

**Review by maxpumperla — CHANGES_REQUESTED** [review]
lgtm, pending the two missing imports

File: rllib/__init__.py (hunk 1 above)
Comment (sven1977) [reply rllib/__init__.py:60]: Oh, sorry, this shouldn't be here. Will remove ...

**Review by maxpumperla — APPROVED** [review]

**Pull request merged**